x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlRealmTests.java

```diff
@@ -337,14 +337,15 @@ public void testAbsurdlyLowMinimumRefreshThrowsException() throws Exception {
     }
```

Comment:
Optional: might also test if we don't get any roles resolved in the first place (i.e., if delegate.resolveRoles return empty)